build_rocm_python3

```diff
@@ -26,7 +26,7 @@ done
 shift "$((OPTIND-1))"
 
 # First positional argument (if any) specifies the ROCM_INSTALL_DIR
-ROCM_INSTALL_DIR=/opt/rocm-6.2.0
+ROCM_INSTALL_DIR=$(realpath /opt/rocm)
```

Comment:
this is a good improvement! could you also add it to our develop-upstream, r2.18, r2.17, r2.16, and r2.15, please? Thanks!